File: jv80/JV80.swift

public enum JV80 {
  
  static let editorTruss = JV8X.editorTruss("JV-80", global: Global.patchWerk, perf: Perf.patchWerk, voice: Voice.patchWerk, rhythm: Rhythm.patchWerk, voiceBank: Voice.bankWerk, perfBank: Perf.bankWerk, rhythmBank: Rhythm.bankWerk)
  
  static let sections = JV8X.sections(global: Global.Controller.ctrlr(), perf: Perf.Controller.ctrlr(), hideOut: true)
  public static let moduleTruss = JV8X.moduleTruss(editorTruss, subid: "jv80", sections: sections)
  
}

File: jv80/jv80.js
const JV8X = require('./jv8x.js')
const GlobalController = require('./jv80_global_ctrlr.js')
const PerfController = require('./jv80_perf_ctrlr.js')

const editor = JV8X.editorTruss("JV-80", {
  global: {
    size: 0x21,
    extraParms: [],
    initFile: "jv880-global",
  },
  voice: {
    size: 0x73,
    extraParms: [],
  },
  perf: {
    size: 0x22,
    extraParms: [],
  },
  rhythm: {
    size: 0x33,
    extraParms: [],
  },
})

const moduleTruss = JV8X.moduleTruss(editor, "jv80", GlobalController, PerfController, true)
